spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RestControllerAdvice.java

```diff
@@ -135,7 +136,10 @@ public VndErrors onNotFoundException(Exception e) {
 	 * Log the exception message at warn level and stack trace as trace level.
 	 * Return response status HttpStatus.BAD_REQUEST (400).
 	 */
-	@ExceptionHandler({MissingServletRequestParameterException.class})
+	@ExceptionHandler({
+		MissingServletRequestParameterException.class,
```

Comment:
I think I had a merge conflict here as you probably didn't PR from latest master.